tests/e2e/contracts/multi_file_contract/__init__.py

```
@@ -0,0 +1,20 @@
+from genlayer import *
```

Ruff (0.11.9)

`from genlayer import *` used; unable to detect undefined names

(F403)